pages/verification/index.tsx

```diff
@@ -0,0 +1,169 @@
+import { BaseContainer } from "@components/Page/BaseContainer";
+import { Helmet } from "react-helmet";
+import React, { useState } from "react";
+import LayoutContainer from "@components/LayoutContainer";
+import api from "@shared/fetcher";
+import apiAnonymous from "@shared/fetcherAnonymous";
+import { useCheckExchangeTokenMutation } from "@services/auth/authHooks";
+import { useRouter } from "next/router";
+import { GetServerSidePropsContext } from "next";
+import Cookies from "js-cookie";
+import { ITokenEnum, MessageEnum, UserEnum } from "@shared/enum";
+import { useInitChatAnonymousMutation } from "@services/initChatAnonymous/initChatAnonymousHooks";
+import { toast } from 'react-toastify'
+import getConfig from "next/config";
+import { useGenerateAnonUserInfoMutation } from "@services/generateAnonUserInfo/generateAnonUserInfoHooks";
+import { LoaderWrapper } from "@components/LoaderWrapper";
+
+interface PageProps {
+    exchangeToken?: string
+    isSendMessage?: boolean
+    isFailedVerify?: boolean
+}
+
+const { publicRuntimeConfig } = getConfig();
+
+export default function Verification(props: PageProps) {
+    const router = useRouter();
+    const exchangeToken = useCheckExchangeTokenMutation();
+    const generateAnonUser = useGenerateAnonUserInfoMutation();
+    const initChatAnon = useInitChatAnonymousMutation();
+    const [isLoading, setIsLoading] = useState(props.exchangeToken ? true : false)
+    const getLoginPage = () => {
+        fetch('/api/getWebLogin').then(async (res) => {
+            toast('Sending you to humanID for verification', {
+                autoClose: false,
+                type: 'info',
+            })
+            const data = await res.json();
+            if (data.data?.webLoginUrl) {
+                router.push(data.data?.webLoginUrl)
+            }
+        })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This function `getLoginPage` makes an API call but does not handle potential errors. If the fetch fails for any reason (network issues, server down, etc.), it will throw an unhandled promise rejection. It's recommended to add a `.catch` block to handle these potential errors.
```diff
        fetch('/api/getWebLogin').then(async (res) => {
            toast('Sending you to humanID for verification', {
                autoClose: false,
                type: 'info',
            })
            const data = await res.json();
            if (data.data?.webLoginUrl) {
                router.push(data.data?.webLoginUrl)
            }
        })
+        .catch((error) => {
+            console.error('Failed to fetch web login URL:', error);
+            // Display a user-friendly message here
+        });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->